npm install errors

### Description:

Installing latest with npm gives the following error

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! formBuilder@3.0.1 preinstall: `git submodule update --init --recursive`

npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the formBuilder@3.0.1 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.


### Environment Details:

 - formBuilder Version: 
 - Browser: 
 - OS: 

### Expected Behavior

### Actual Behavior

### Steps to Reproduce

### Screenshot - *(optional)*

--- a/package.json
+++ b/package.json
@@ -91,7 +91,6 @@
     "docs": "mkdocs build",
     "font": "node tools/run icon-font",
     "lint": "eslint ./src --ext .js || true",
-    "preinstall": "git submodule update --init --recursive",
     "semantic-release": "semantic-release",
     "start": "yarn build:vendor && webpack-dev-server --mode development --open --config tools/webpack.config",
     "test": "yarn build",

